src/test/scala/org/allenai/deep_qa/data/OmnibusDaDatasetReaderSpec.scala

```diff
@@ -21,11 +23,54 @@ class OmnibusDaDatasetReaderSpec extends FlatSpecLike with Matchers {
   val questionText2 = "Coal is a nonrenewable energy resource. People burn coal to " +
     "make electricity and heat buildings. Explain how long it would most likely take for new coal " +
     "to form."
+  val transformedQuestionText2 = "Coal is a nonrenewable energy resource. People burn coal to " +
+    "make electricity and heat buildings. How long it would most likely take for new coal to form."
   val answerText2 = "Coal may slowly form over millions of years."
   val row2 = s""""1",,"false","${questionText2}","${answerText2}",""" +
     """"1","false","false","false","Source2","5","2015","","Tag2",""" +
     """"legacyId2","importedId2""""
 
+  val questionText3 = "Read the given statement and determine if the behavior is "+
+    "either learned or innate. A bear eating salmon caught from a river."
+  val transformedQuestionText3 = "Read the given statement and determine if the "+
+    "behavior is either learned or innate. A bear eating salmon caught from a river."
+
+  val questionText4 = "Even though an adult amphibian can live on land, why must it " +
+    "return to the water? (Explain.)"
+  val transformedQuestionText4 = "Even though an adult amphibian can live on land, " +
+    "why must it return to the water? (Explain.)"
+
+  // TODO(nelson): Would like to be able to handle plural/singular.
+  val questionText5 = "Name four organisms you might find living in or around a " +
+    "freshwater lake."
+  val transformedQuestionText5 = "What is four organisms you might find living in or around a " +
+  "freshwater lake."
+  val questionText6 = "List the conditions that must be present in order for a " +
+  "seed to germinate."
+  val transformedQuestionText6 = "What is the conditions that must be present in "+
+  "order for a seed to germinate."
+
+  // TODO(nelson): not clear how to insert the proper verbs here.
```

Comment:
You want to rotate the dependencies in the dependency tree, moving the subject "most animal life on Earth" after the auxiliary verb "could not".  But being able to do that correctly assumes a correct dependency parse, which we don't have.  So we should just punt on this, unless you want to write regexes over POS tags...